parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 NODE_INDEX=$(hostname | tail -c 2)
 NODE_NAME=$(hostname)
-PRIVATE_IP=$(hostname -I | cut -d' ' -f1)
```

Comment:
This var assignment has been moved into the `cse_helpers.sh` instead for more general purpose usage, and the etcd vars are moved down to local vars inside the funcs that use them